scripts/on-premises/single-server/cvdr_create_multiple_hosts.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    echo "Usage: $0 [--num_hosts <num_hosts>] -- [<subcommands of cvdr create>]"
+    echo "Prerequisites:"
+    echo "  - Prepare accessible Cloud Orchestrator server"
+    echo "  - Install cuttlefish-cvdremote to obtain cvdr binary"
+    echo "  - Prepare a proper configuration file of cvdr"
+}
+
+NUM_HOSTS=1
+while true; do
+    case "$1" in
+        -n|--num_hosts)
+            NUM_HOSTS="$2"
+            shift 2
+            ;;
+        -h|--help)
+            usage
+            exit 0
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            echo "Invalid command" >&2
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+echo "Host 1 creation started"
+cvdr create $@
```

Comment:
`$@` should be in quotes: `"$@"`. Without quotes, any argument that has a space in it will be passed to `cvdr create` as different arguments.